diff --git a/node_modules/react-native-google-mobile-ads/lib/typescript/specs/modules/NativeGoogleMobileAdsNativeModule.d.ts b/node_modules/react-native-google-mobile-ads/lib/typescript/specs/modules/NativeGoogleMobileAdsNativeModule.d.ts
index e629ddc..f0b969f 100644
--- a/node_modules/react-native-google-mobile-ads/lib/typescript/specs/modules/NativeGoogleMobileAdsNativeModule.d.ts
+++ b/node_modules/react-native-google-mobile-ads/lib/typescript/specs/modules/NativeGoogleMobileAdsNativeModule.d.ts
@@ -30,7 +30,12 @@ export type NativeAdEventPayload = {
 export interface Spec extends TurboModule {
     load(adUnitId: string, requestOptions: UnsafeObject): Promise<NativeAdProps>;
     destroy(responseId: string): void;
-    readonly onAdEvent: EventEmitter<NativeAdEventPayload>;
+    readonly onAdEvent: (
+        eventType: string,
+        listener: (event: NativeAdEventPayload) => void
+    ) => void;
+    // readonly onAdEvent: EventEmitter<NativeAdEventPayload>;
+    // onAdEvent: (event: string) => void;
 }
 declare const _default: Spec;
 export default _default;
diff --git a/node_modules/react-native-google-mobile-ads/src/specs/modules/NativeGoogleMobileAdsNativeModule.ts b/node_modules/react-native-google-mobile-ads/src/specs/modules/NativeGoogleMobileAdsNativeModule.ts
index 46c98ec..0934272 100644
--- a/node_modules/react-native-google-mobile-ads/src/specs/modules/NativeGoogleMobileAdsNativeModule.ts
+++ b/node_modules/react-native-google-mobile-ads/src/specs/modules/NativeGoogleMobileAdsNativeModule.ts
@@ -58,7 +58,12 @@ export type NativeAdEventPayload = {
 export interface Spec extends TurboModule {
   load(adUnitId: string, requestOptions: UnsafeObject): Promise<NativeAdProps>;
   destroy(responseId: string): void;
-  readonly onAdEvent: EventEmitter<NativeAdEventPayload>;
+  readonly onAdEvent: (
+    eventType: string,
+    listener: (event: NativeAdEventPayload) => void
+  ) => void;
+  // readonly onAdEvent: EventEmitter<NativeAdEventPayload>;
+  // onAdEvent: (event: string) => void;
 }
 
 export default TurboModuleRegistry.getEnforcing<Spec>('RNGoogleMobileAdsNativeModule');
